chain/vm/vm.go

```diff
@@ -56,12 +56,19 @@ func ResolveToKeyAddr(state types.StateTree, cst cbor.IpldStore, addr address.Ad
 		return address.Undef, xerrors.Errorf("failed to find actor: %s", addr)
 	}
 
+	if state.Version() >= types.StateTreeVersion5 {
+		if act.Address == nil {
+			return address.Undef, xerrors.Errorf("actor doesn't have expected address: %s", addr)
```

Comment:
The error refers to lacking a _predictable_ address, right? Also switching the order for a clearer message.

```suggestion
			return address.Undef, xerrors.Errorf("actor at %s doesn't have a predictable address", addr)
```

Reply:
yar, will fix.

Reply:
done.